microprofile-fault-tolerance-quickstart/src/test/java/org/acme/faulttolerance/CoffeeResourceIT.java

```diff
@@ -1,35 +1,22 @@
 package org.acme.faulttolerance;
 
-import static io.restassured.RestAssured.get;
-import static org.hamcrest.CoreMatchers.hasItem;
-import static org.hamcrest.CoreMatchers.is;
-
 import io.quarkus.test.junit.NativeImageTest;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 @NativeImageTest
-public class CoffeeResourceIT {
+public class CoffeeResourceIT extends CoffeeResourceTest {
 
     @Test
-    public void testAvailability() {
-        get("/coffee/1/availability").then()
-                .statusCode(200).body(is("20"));
-        get("/coffee/1/availability").then()
-                .statusCode(200).body(is("20"));
-        get("/coffee/1/availability").then()
-                .statusCode(500).body(is("RuntimeException: Service failed."));
-        get("/coffee/1/availability").then()
-                .statusCode(500).body(is("RuntimeException: Service failed."));
-        get("/coffee/1/availability").then()
-                .statusCode(500).body(is("CircuitBreakerOpenException: getAvailability"));
+    @Override
+    @Disabled("@Inject in tests doesn't work for native mode")
```

Comment:
Why not using the `DisabledOnNativeImage` annotation?

Reply:
Because I wasn't aware of it ;) Will update the PR